sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -54,34 +55,32 @@
       "ky",
       "E"
     ],
-    "slow_axes": "@attrs:metadata/instrument/analyzer/slow_axes",
+    "slow_axes": "t.b.d.",
     "energy_resolution": {
-      "resolution": "@attrs:metadata/instrument/analyzer/energy_resolution",
+      "resolution": 100,
       "resolution/@units": "meV",
       "physical_quantity": "energy",
       "type": "estimated"
     },
     "momentum_resolution": {
-      "resolution": "@attrs:metadata/instrument/analyzer/momentum_resolution",
+      "resolution": 0.08,
       "resolution/@units": "1/angstrom",
       "physical_quantity": "momentum",
       "type": "estimated"
     },
     "spatial_resolution": {
-      "resolution": "@attrs:metadata/instrument/analyzer/spatial_resolution",
+      "resolution": 1,
       "resolution/@units": "µm",
       "physical_quantity": "length",
       "type": "estimated"
     }
   },
   "/ENTRY[entry]/INSTRUMENT[instrument]/ELECTRONANALYSER[electronanalyser]/COLLECTIONCOLUMN[collectioncolumn]": {
     "projection": "@attrs:metadata/scientificMetadata/Collection/projection",
-    "scheme": "None",
+    "scheme": "momentum dispersive",
     "lens_mode": "@attrs:metadata/scientificMetadata/Collection/lens_mode",
     "extractor_voltage": "@attrs:metadata/scientificMetadata/Collection/extractor_voltage",
     "extractor_voltage/@units": "V",
-    "extractor_current": "t.b.d.",
-    "extractor_current/@units": "µA",
```

Comment:
If this is not available right now, just remove.

Reply:
For the moment this is tracked via Aux channels together with extractor voltage, thus we didn't send to SciCat but in principal this can be added into SciCat

Reply:
You can get it then during metadata collection from the dataframe.